smlar.c

```diff
@@ -36,6 +41,7 @@ getDefaultOpclass(Oid amoid, Oid typid)
 	HeapTuple	tuple;
 	Relation	heapRel;
 	Oid			opclassOid = InvalidOid;
+	Form_pg_proc form;
```

Comment:
This needs a PG12 ifdef around it too.

Reply:
HEy @systemed , thanks for your help!
Is this what you had in mind?
Best,
Remi

Reply:
Spot on!